examples/check-expression/check_expression.bal

```diff
@@ -1,18 +1,23 @@
 import ballerina/io;
 
-// Convert `bytes` to a `string` value and then to an `int` value.
-function intFromBytes(byte[] bytes) returns int|error {
-
-    // Use `check` with an expression that may return `error`.
-    // If `string:fromBytes(bytes)` returns an `error` value, `check`
-    // makes the function return the `error` value here.
-    // If not, the returned `string` value is used as the value of the `str` variable.
+function intFromBytesWithCheck(byte[] bytes) returns int|error {
     string str = check string:fromBytes(bytes);
-
     return int:fromString(str);
 }
 
+// Same as `intFromBytesWithCheck` but with explicit error handling.
+function intFromBytesExplicit(byte[] bytes) returns int|error {
+    string|error res = string:fromBytes(bytes);
+    // Handling the error explicitly.
```

Comment:
Would it make sense to switch the two? I would also make `intFromBytesExplicit` just `intFromBytes`.

Reply:
Fixed